Felienne/hedy
# [BUG FIX] Correctly resets customizations on removal

**Description**
In this PR we fix a small bug where the customizations page would incorrectly reset on customizations removal. Of course, simply re-loading the page would also have done the trick. But this nicer as it doesn't require a re-load so we are still able to show the feedback message.

**Fixes**
This PR fixes #2560.

**How to test**
Make sure you are a teacher and have some class customizations. Go to the customizations page and remove them, verify that all levels and adventures are correctly checked after the removal. Also verify that all teacher adventures are unchecked as well as all additional settings.
